src/runtime/recipe/recipe.ts

```diff
@@ -697,6 +698,16 @@ export class Recipe implements Cloneable<Recipe> {
     }
     return slot;
   }
+
+  get isLongRunning(): boolean {
```

Comment:
Maybe we could have ephemeral vs. persistent if we have to put something in code? That's a better name IMO than long-running. And with stores of long-runnning arcs being instantiated eagerly, they are kind of like "persistent" arcs. WDYT? @shans

Reply:
I'm happy to rename in general, but "persistent" would be confusing, because `persistent` capability means on disk, but long running arcs could use in-memory storage as well.